src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,61 @@
+
+## Spider Clan Code
+
+All Space Ninjas will now have a Spider Clan Code they will have to live by, and this thus affects gameplay substantially by imposing certain restrictions.
+- The already existing one, being the inability to shoot ranged weapons, will be included in this Code.
+- The second restriction is making it so the Space Ninjas can no longer tend to their own wounds - This includes applying topicals, injecting syringes, consuming pills, and drinking liquid chemicals, as it would be preferred that they are not even spotted during their mission. To compensate, Space Ninjas will now have increased passive regeneration as well as faster bloodloss recovery that far exceeds what normal crewmates could ever achieve due to their bio-engineered bodies.
+
+## New Items & Mechanics
+
+### Flavour Additions
+
+- The Space Ninja now comes with an adequately themed satchel, as well as a katana sheath.
+- Equipping the space ninja helmet is now an instantaneous toggle action that comes with the space ninja suit.
+
+### Ability Additions
+- The Space Ninja can now fabricate ninja caltrops, a variant of ninja throwing stars with an emphasis on piercing damage and bleeding, once deployed, that is, dropped on the ground, they will progressively get more and more camouflaged to the point of near-invisibility. Upon being stepped on, the caltrops will stick to the victim and perform continous bloodloss damage until taken off.
+- The ninja visor can now alternate between normal vision and night vision.
+
+### Objective Mechanics
+
+- All stealing objectives can now only be performed with a specialized item that is dropped by the ninja upon death, unlike previously where they only used their gloves - This allows the crew to regain their losses.
+- A strongbox for Cargo containing all their deposited spesos, can be the target of one of the Space Ninja's objectives, located in the vault - An alternative to "Steal X technologies".
+- An AI law board that can be used for one of the Space Ninja's objectives - An alternative to "Detonate your spider charge".
+- The criminal records computer now only announces the error locally in the Security channel, instead of it being broadcasted as a Central Command announcement station-wide.
+- The communications computer objective now revolves around the Space Ninja sending a Spider Clan Announcement from it, the contents of said announcement being up to their discretion. Life before, it requires the use of turned on space ninja gloves, the one key difference being that now the EMAG interaction is instantaneous, overriding all access restrictions as well as unlocks Spider Clan as an announcement sender.
+
+## Balance Changes
+
+### Nerfs
+- [Fix ninja stunlocking](https://github.com/space-wizards/space-station-14/pull/33244)
+- All Space Ninjas are now bio-engineered humans, making it so they don't get specie-specific attributes such as tail-dragging.
```

Comment:
I'd like to still see player characters, I'm partially against having everyone be human for the sake of balance, even though it's easiest. I know I love being a spider as a ninja just because I'm part of the "Spider Clan" and I RP it as well.

Reply:
> I'd like to still see player characters, I'm partially against having everyone be human for the sake of balance, even though it's easiest. I know I love being a spider as a ninja just because I'm part of the "Spider Clan" and I RP it as well.

I disagree, while part of it is for the sake of balance, I find it to also be much more immersive for certain factions to prefer certain species, just like how S.E.L.F. would prefer IPCs, the Spider Clan prefers bio-engineered humans, and honestly, the ninja fit just goes much harder when there's no added characteristics like wings or tails.

Reply:
B-b-b-but... spider... spider clan...